### PR TITLE
[#11470] Statuses issues

### LINKED ISSUE #11470: Statuses issues
# Bug Report

1) Outgoing reaction alignment is wrong
image

2) statuses page is flickering when there are images 

3) swap profile and statuses tabs

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #11470

### CODE REVIEW

File: src/status_im/ui/screens/chat/message/message.cljs [hunk 1]
```diff
@@ -234,24 +234,29 @@
...
-        uri        (:image content)]
+  (let [dimensions (reagent/atom [image-max-width image-max-height])
+        visible (reagent/atom false)
+        uri (:image content)]
     (react/image-get-size
      uri
      (fn [width height]
-       (reset! dimensions [width height])))
+       (when (< width height)
+         (let [k (/ height image-max-height)]
```
Comment (cammellos) [inline src/status_im/ui/screens/chat/message/message.cljs]: It'd be good to either move to a separate function, or add a comment explaining what it's doing, as not immediately clear

File: src/status_im/ui/screens/status/views.cljs [hunk 2]
```diff
@@ -25,32 +25,34 @@
...
 
 (defn message-content-image [uri _]
-  (let [dimensions (reagent/atom [image-max-dimension image-max-dimension])]
+  (let [dimensions (reagent/atom [nil image-max-dimension])]
     (react/image-get-size
      uri
      (fn [width height]
-       (let [k (/ (max width height) image-max-dimension)]
-         (reset! dimensions [(/ width k) (/ height k)]))))
+       (when (< width height)
```
Comment (cammellos) [inline src/status_im/ui/screens/status/views.cljs]: Similarly, it'd be nice to have a comment here as well

**Review by cammellos — APPROVED** [review]

**Comment by flexsurfer** [comment]
fyi @errorists

**Comment by qoqobolo** [comment]
@flexsurfer Q: is it expected that there is still "`Fetch more messages`", rather than just “`Fetch more`”? Ask because of this comment https://github.com/status-im/status-react/issues/11470#issuecomment-733555912

Screenshot 2020-12-01 at 11 09 53

**Comment by flexsurfer** [comment]
thanks, it's not expected, looking

**Comment by flexsurfer** [comment]
fixed

**Comment by qoqobolo** [comment]
Thank you!
And another question about borders. There seems to be some space between the border and the image. Wdyt? Can this be fixed?
photo_2020-12-01_11-35-38

**Comment by flexsurfer** [comment]
fixed, thanks

**Comment by qoqobolo** [comment]
### ISSUE 3: Statuses page is flickering when there are images (Android)

ezgif com-gif-maker (10)

**Comment by flexsurfer** [comment]
@qoqobolo did this happen after my latest changes ? or it wasn't fixed in this PR, because i tested on Android and it worked good

**Comment by flexsurfer** [comment]
@qoqobolo it seems you're trying the wrong version, there is no blue background in this PR

**Comment by qoqobolo** [comment]
@flexsurfer Thanks for fixes! Looks great now.
Ready to merge.

**Pull request merged**